### ISSUE TITLE
test.py result problem

### ISSUE THREAD

**Issue by Bigtuo** [cuiziteng/ICCV_MAET#1]
1. There is a small problem with data preprocessing
2. On the Exdark data set, the test result is relatively high, which is far from the paper:
+-----------+------+------+--------+-------+
| class | gts | dets | recall | ap |
+-----------+------+------+--------+-------+
| Bicycle | 218 | 653 | 0.991 | 0.963 |
| Boat | 275 | 884 | 0.956 | 0.891 |
| Bottle | 315 | 1216 | 0.949 | 0.905 |
| Bus | 138 | 301 | 0.993 | 0.984 |
| Car | 603 | 1557 | 0.970 | 0.933 |
| Cat | 175 | 450 | 0.971 | 0.948 |
| Chair | 440 | 1926 | 0.927 | 0.871 |
| Cup | 330 | 1029 | 0.967 | 0.938 |
| Dog | 200 | 458 | 0.990 | 0.967 |
| Motorbike | 218 | 758 | 0.982 | 0.920 |
| People | 1459 | 3840 | 0.966 | 0.911 |
| Table | 299 | 1860 | 0.913 | 0.788 |
+-----------+------+------+--------+-------+
| mAP | | | | 0.918 |
+-----------+------+------+--------+-------+
{'mAP': 0.9183435440063477}

**Comment by Bigtuo** [comment]
> Hello, I've upload the baiduyun link of the EXDark dataset in VOC format (see Readme.md), you could directly download and use it.

Hello, I used exdrak data to retrain, and the results are basically the same, but I used the data set I collected to train with very low AP. What is the problem? How can I solve it?

**Comment by cuiziteng** [comment]
Maybe you could try different epoch numbers, learning rate, or anchor size, to adjust on your own dataset.

**Comment by Bigtuo** [comment]
Hello, would you like to know the pre-training model epoch273.pth here is only applicable to yolov3?

**Comment by cuiziteng** [comment]
是的，我后续会提供DETR的模型，请持续关注。

**Comment by Bigtuo** [comment]
好的谢谢



---Original---
From: ***@***.***&gt;
Date: Mon, Oct 18, 2021 18:51 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [cuiziteng/ICCV_MAET] test.py result problem (#1)




 
是的，我后续会提供DETR的模型，请持续关注。
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
